staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal_test.go

```diff
@@ -155,8 +156,11 @@ func TestEnsureInternalLoadBalancer(t *testing.T) {
 
 	gce, err := fakeGCECloud(vals)
 	require.NoError(t, err)
-
 	svc := fakeLoadbalancerService(string(LBTypeInternal))
+	svc, err = gce.client.CoreV1().Services(svc.Namespace).Create(svc)
```

Comment:
Why do we create this service twice
What is createInternalLoadBalancer() doing that we need to call create again here

Reply:
The service is created here so that it can be looked up with a services "Get" command. This is needed so that the code in createInternalLoadBalancer can lookup and patch the service. That code does not create the k8s service again.

Reply:
Looking at the test code, it's pretty confusing why this is done?

Reply:
I modified the tests so that creation of k8s service happens within a util function. PTAL.

Reply:
discussed offline, we will leave the test code to create the services inline.